Vue/src/assets/main.css

```diff
@@ -0,0 +1,8 @@
+.main {
+  font-family: Avenir, Helvetica, Arial, sans-serif;
+  -webkit-font-smoothing: antialiased;
+  -moz-osx-font-smoothing: grayscale;
+  color: #2c3e50;
+  margin: 50px 50px;
+  width: 90vh;
```

Comment:
it's better to correct this part to use `vw` instead of `vh`

Reply:
Turns out this style applied everywhere from the get go. Fixed it.